docs/api/ref/Errors.md

```diff
@@ -346,7 +379,7 @@ Contains constants representing the errors that can be encountered when calling
 
 ## `BridgingError`
 
-`@experimental`
+[`@experimental`](../../fdc3-compliance#experimental-features)
```

Comment:
spelling mistake (in docs that have not been altered but I thought I'd mention it).

```
Returned if a Desktop Agent did not return a response, via Desktop Agent Bridging, within the alloted timeout.
```

should be 

```
Returned if a Desktop Agent did not return a response, via Desktop Agent Bridging, within the **allotted** timeout.
```

Reply:
fixed